src/views/TabView/TabBarBottom.js

```diff
@@ -220,17 +248,29 @@ class TabBarBottom extends React.PureComponent<Props> {
                     ? onPress({ previousScene, scene, jumpToIndex })
                     : jumpToIndex(index)}
               >
-                <Animated.View
-                  style={[
-                    styles.tab,
-                    isLandscape && useHorizontalTabs && styles.tabLandscape,
-                    !isLandscape && useHorizontalTabs && styles.tabPortrait,
-                    { backgroundColor },
-                    tabStyle,
-                  ]}
-                >
-                  {this._renderIcon(scene)}
-                  {this._renderLabel(scene)}
+                <Animated.View style={[styles.tab, { backgroundColor }]}>
+                  <View
+                    onLayout={evt => {
+                      const { useHorizontalTabs, viewWidth } = this.state;
+                      if (
+                        viewWidth > 0 &&
+                        useHorizontalTabs &&
+                        viewWidth - 20 <
+                          evt.nativeEvent.layout.width * routes.length
```

Comment:
Can you add some comments as to what you're doing here and why? For example, where is that 20 coming from?